INSTALL.md

```diff
@@ -249,11 +249,17 @@ filling in this section of the README.
 
 ### Linux / Windows / BSD
 
+Running `build` will output the binary in `target/release/`
 ```sh
 cargo build --release
 ```
 
-On Linux/BSD, if it is desired to build Alacritty without support for either the
+Alternatively you can use `install` that will install it globally (in `~/.cargo/bin` by default)
+```
+cargo install --path alacritty 
+```
+
+On Linux/BSD, if it is desired to build (or install) Alacritty without support for either the
```

Comment:
I'm not sure about this one though. Since we tell where the binary is, and users can copy it if they want.

Telling to do an `install` like that is a bit wrong, simply because we have `features` and missing them in `install` will restart the build without those features. This command with `install` is also not really correct, because you need to pass `--locked` or `--frozen` for installs in general.